### ISSUE TITLE
Produce  macro-averaged F1-score (MF1) results

### ISSUE THREAD

**Issue by NithyasreeVP** [emadeldeen24/TS-TCC#13]
Hi!

In the paper (Results section), you mentioned that the performance is evaluated using two metrics: accuracy and the macro-averaged F1-score (MF1). But when I run main.py, I get only accuracy metric results.
'''
total_acc.append(labels.eq(predictions.detach().argmax(dim=1)).float().mean())
'''
 Can you let me know how MF1 is calculated as well?

Thanks in advance.

**Comment by emadeldeen24** [comment]
Hi,
you get them eventually in the classification report.
If you wish to see the F1-score at each run, you can check our implementation [here](https://github.com/emadeldeen24/eval_ssl_ssc/blob/2c7a3b789ee97ded179ba6254276a6738f756559/trainer.py#L226).

**Comment by NithyasreeVP** [comment]
Thank you!
